RocketChat/Rocket.Chat
# [IMPROVE] Rewrite URL check modal

## Proposed changes (including videos or screenshots)


![image](https://user-images.githubusercontent.com/40830821/124157878-a3d80380-da6f-11eb-8bd8-03dffd14c658.png)



## Issue(s)


## Steps to test or reproduce

Enter your local server with a different URL configured.
Ex.: If your server is configured to use `http://localhost:3000`, access through `http://0.0.0.0:3000`.

The modal should show up, and the setting updated to the current URL you're accessing through (if confirmed)
## Further comments
